### PR TITLE
UF-245: use of resource filtering in order to avoid hard version number

### PR DESCRIPTION
**Pull request by porcelli**

### CODE REVIEW

File: uberfire-archetype/uberfire-project-archetype/src/main/resources/archetype-resources/__rootArtifactId__-parent-with-dependencies/pom.xml [hunk 1]
```diff
@@ -3,12 +3,12 @@
 
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>${groupId}</groupId>
-    <artifactId>${rootArtifactId}</artifactId>
-    <version>${version}</version>
+    <groupId>\${groupId}</groupId>
```
Comment (manstis) [inline uberfire-archetype/uberfire-project-archetype/src/main/resources/archetype-resources/__rootArtifactId__-parent-with-dependencies/pom.xml]: Why the leading ```\```?

Comment (porcelli) [inline uberfire-archetype/uberfire-project-archetype/src/main/resources/archetype-resources/__rootArtifactId__-parent-with-dependencies/pom.xml]: escaping (line 71 from the above file)

**Comment by manstis** [comment]
Once I understand the prolific use of a proceeding ```\``` I am happy to +1.

**Comment by manstis** [comment]
+1 to merge.

**Comment by csadilek** [comment]
Thanks.

master: 05d224718881dfcfd786988def738ffe2baf7be6

**Comment by psiroky** [comment]
Thanks @porcelli. This will ease the version updates and thus releasing as well!

**Pull request closed without merging**